Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/core/PersonCertificates.kt

```diff
@@ -9,7 +9,9 @@ data class PersonCertificates(
     val certificates: List<CwaCovidCertificate>,
     val isCwaUser: Boolean = false,
     val badgeCount: Int = 0,
-    val dccWalletInfo: DccWalletInfo? = null
+    val dccWalletInfo: DccWalletInfo? = null,
+    val hasBooster: Boolean = false,
+    val hasDccReissuance: Boolean = false,
```

Comment:
Maybe make these 2 property names more descriptive like `hasBoosterBadge` and `hassDccReissuance` badge?